pkgs/development/python-modules/tempest/default.nix

```diff
@@ -0,0 +1,88 @@
+{ lib
+, buildPythonApplication
+, fetchPypi
+, pbr
+, cliff
+, jsonschema
+, testtools
+, paramiko
+, netaddr
+, oslo-concurrency
+, oslo-config
+, oslo-log
+, stestr
+, oslo-serialization
+, oslo-utils
+, fixtures
+, pyyaml
+, subunit
+, stevedore
+, prettytable
+, urllib3
+, debtcollector
+, unittest2
+, hacking
+, oslotest
+, bash
+, python3
+}:
+
+buildPythonApplication rec {
+  pname = "tempest";
+  version = "28.0.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "24fcc0baa2044454b17b6b4aa2b1b19682cf95eb92ca38a2f289d3cbc488b170";
+  };
+
+  propagatedBuildInputs = [
+    pbr
+    cliff
+    jsonschema
+    testtools
+    paramiko
+    netaddr
+    oslo-concurrency
+    oslo-config
+    oslo-log
+    stestr
```

Comment:
Removing stestr from checkInputs did not actual work.. Sorry about that it need to be re-added.

Reply:
I should have noticed it. Sorry about that too.